### PR TITLE
[#26] pgmoneta_ext_promote()

### PR DESCRIPTION
**Pull request by GuChad369**
Define a custom extension function `pgmoneta_ext_promote()` to implement the functionality of `pg_promote()`.

### CODE REVIEW

File: src/pgmoneta_ext/lib.c [hunk 1]
```diff
@@ -472,6 +473,32 @@ pgmoneta_ext_receive_file_chunk(PG_FUNCTION_ARGS)
...
+      promoted = DirectFunctionCall2(pg_promote,
+                                     BoolGetDatum(false),
+                                     Int32GetDatum(0));
+
+      PG_RETURN_BOOL(DatumGetBool(promoted));
+   }
+   else
+   {
+      ereport(LOG, errmsg_internal("pgmoneta_ext_promote: Current role is not a superuser"));
+      PG_RETURN_NULL();
```
Comment (jesperpedersen) [inline src/pgmoneta_ext/lib.c]: We need `false` here

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
Did you rebase ?

**Comment by GuChad369** [comment]
yeah

**Comment by jesperpedersen** [comment]
Merged. (I was in the wrong shell window)

Thanks for your contribution !

Could you take add a function to pgmoneta's `extension.h` file as well ?

**Comment by jesperpedersen** [comment]
https://github.com/pgmoneta/pgmoneta/issues/428

**Pull request merged**